clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -545,6 +545,21 @@ public void testUpdateFetchPositionDisconnect() {
     }
 
     @Test
+    public void testUpdateFetchPositionOfPausedPartitions() {
```

Comment:
Could we have one more test case similar to this, but in which the paused partition already has a position? This verifies that `updateFetchPositions` does not overwrite the current position.